src/backend/main.ts

```diff
@@ -301,6 +301,9 @@ if (!gotTheLock) {
     handleProtocol(argv)
   })
   app.whenReady().then(async () => {
+    const migrationComponent = new MigrationComponent()
+    await migrationComponent.applyMigrations()
+
```

Comment:
This is a little awkward, and is a result of porting this to "non-components-system" Heroic. If components would be in main right now, you'd not see any of this here (the component would instead be initialized automatically)

Reply:
I have some questions about the components, but there's no PR for that right? to not ask the questions here

Reply:
There's no PR yet, no. I do have a WIP branch, but it's a little outdated
The reason I haven't made a PR yet is that to properly show what components can do (and to get them some actual use, instead of just being an unused system), I need some preliminary steps (which this PR is one of)

Reply:
I ended up uncoupling this from the "components" model now. I'm not sure whether I'll continue with components, at least for now. Lots of other things to do first